docs/gameplay-multiplayer-realtime.md

```diff
@@ -319,7 +319,7 @@ The binary content in each data message should be as __small as possible__. It i
 var id = "<matchid>";
 var opCode = 1;
 var data = { "move": {"dir": "left", "steps": 4} };
-socket.send({ match_data_send: { match_id: id, op_code: opCode, data: data } });
+socket.sendMatchState(id, opCode, data);
```

Comment:
Should we document the `presence` field here too?

Reply:
I think so, but I actually noticed something odd:

https://github.com/heroiclabs/nakama-js/blob/master/src/socket.ts#L221
https://github.com/heroiclabs/nakama-js/blob/master/src/socket.ts#L308

The current JS SDK `sendMatchState` only accepts one presence rather than multiple, which doesn't quite make sense to me. Should I fix? @novabyte